subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -115,3 +121,29 @@ def enabled = providers.systemProperty("some-property").forUseAtConfigurationTim
 Support for using configuration caching with certain Gradle features is not yet implemented.
 
 - Composite builds
+
+
+[[testkit]]
+== Testing Build Logic with TestKit
+
+The Gradle TestKit (a.k.a. just TestKit) is a library that aids in testing Gradle plugins and build logic generally.
+For general guidance on how to use TestKit see the <<test_kit.adoc#test_kit,dedicated chapter>>.
+
+To enable configuration caching in your tests, you can pass the `--configuration-cache=on` argument to link:{javadocPath}/org/gradle/testkit/runner/GradleRunner.html[GradleRunner] or use one of the other methods described in <<configuration_cache.adoc#enable,Enabling the configuration cache>>.
+
+You need to run your tasks twice.
+Once to store the configuration cache.
```

Comment:
⭕ I'm a little (just a very little) bothered by the phrase _store the configuration cache_. I understand _cache_ can mean the thing(s) being stored but I'm more used to its meaning as _the place_ where things are stored. Maybe _to **prime** the configuration cache_? I'm really not sure and I'd be curious to hear from a native speaker.

Reply:
I went with _prime_ and requested Adam's review.